core/templates/dev/head/app.js

```diff
@@ -30,9 +30,10 @@ var oppia = angular.module(
     window.GLOBALS ? (window.GLOBALS.ADDITIONAL_ANGULAR_MODULES || [])
                    : []));
 
-// TODO(sll): Get this to read from a common JSON file; it's replicated in
-// feconf.
-oppia.constant('CATEGORY_LIST', GLOBALS.ALL_CATEGORIES);
+for (var constantName in CONSTANTS) {
+  oppia.constant(constantName, CONSTANTS[constantName]);
```

Comment:
Actually, if you can get access to the real constants, it might be better to just use that instead. Is that possible?